lib/ansible/modules/files/acl.py

```diff
@@ -94,50 +94,44 @@
     choices: [ 'yes', 'no' ]
     description:
       - Recursively sets the specified ACL (added in Ansible 2.0). Incompatible with C(state=query).
+
+  no_mask:
+    version_added: "2.4"
+    required: false
+    default: null
+    description:
+      - Added feature to not recalculate the effective rights mask by default (setfacl -n, --no-mask).
+
 author:
     - "Brian Coca (@bcoca)"
     - "Jérémie Astori (@astorije)"
 notes:
     - The "acl" module requires that acls are enabled on the target filesystem and that the setfacl and getfacl binaries are installed.
     - As of Ansible 2.0, this module only supports Linux distributions.
-    - As of Ansible 2.3, the I(name) option has been changed to I(path) as default, but I(name) still works as well.
 '''
 
 EXAMPLES = '''
 # Grant user Joe read access to a file
-- acl:
-    path: /etc/foo.conf
-    entity: joe
-    etype: user
-    permissions: r
-    state: present
+- acl: name=/etc/foo.conf entity=joe etype=user permissions="r" state=present
```

Comment:
Why change the example docs? The yaml dict style is the preferred format for EXAMPLES